asm2/main.js

```diff
@@ -0,0 +1,54 @@
+function openForm() {
+    document.getElementById("myForm").style.display = "block";
+  }
+  
+function closeForm(){
+    document.getElementById("myForm").style.display = "none";
+}
+
+var total = [];
+function onFormSubmit() {
+    var formData = readFormData()
```

Comment:
Should use let instead of var (research)
```suggestion
    let formData = readFormData()
```